### PR TITLE
Fix Windows regresion on OgreLightVisual

### PR DESCRIPTION
**Pull request by ahcorde**
Related with this regression issue https://github.com/ignitionrobotics/ign-rendering/issues/226

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

**Comment by chapulina** [comment]
Weird to see the heightmap warning on Windows, it had been addressed on #180. I think the pragmas can be extended to wrap the whole file.

**Comment by ahcorde** [comment]
@chapulina should I fix this here ?

**Comment by chapulina** [comment]
> @chapulina should I fix this here ?

That would be great, thanks!

**Comment by ahcorde** [comment]
@j-rivero or @chapulina  any idea about this one:

```
D:\Jenkins\workspace\ign_rendering-pr-win\ws\ign-rendering\ogre2\src\Ogre2Light.cc(284,1): fatal error C1001: Internal compiler error. [D:\Jenkins\workspace\ign_rendering-pr-win\ws\build\ignition-rendering5\ogre2\src\ignition-rendering5-ogre2.vcxproj]
```

**Comment by chapulina** [comment]
>  any idea about this one:

That looks very strange, I retriggered the build :thinking:

**Comment by ahcorde** [comment]
@osrf-jenkins retest this please

File: ogre/src/OgreLightVisual.cc [hunk 1]
```diff
@@ -75,7 +75,9 @@ void OgreLightVisual::CreateVisual()
     this->dataPtr->line = std::shared_ptr<OgreDynamicLines>(
       new OgreDynamicLines(MT_LINE_LIST));
     this->ogreNode->attachObject(this->OgreObject());
-    this->dataPtr->line->setMaterial("Default/TransGreen");
+    MaterialPtr lightVisualMaterial =
+      this->Scene()->Material("Default/TransGreen")->Clone();
+    this->SetMaterial(lightVisualMaterial, false);
```
Comment (iche033) [inline ogre/src/OgreLightVisual.cc:80]: is this needed? since we're not modifying the material, we don't need a copy right?

Comment (ahcorde) [reply ogre/src/OgreLightVisual.cc:80]: Fixed https://github.com/ignitionrobotics/ign-rendering/pull/228/commits/e8ea4e72e0d0ad7b9cf7c6e9d951e7f3b0c240b5

**Comment by ahcorde** [comment]
@osrf-jenkins retest this please

**Comment by ahcorde** [comment]
@osrf-jenkins retest this please

**Review by chapulina — APPROVED** [review]
CI is happy and I believe all comments have been addressed. Merging to fix CI.

**Pull request merged**